.NET/Microsoft.Recognizers.Text.DateTime/English/Parsers/EnglishDateParserConfiguration.cs

```diff
@@ -145,6 +145,14 @@ public int GetSwiftDay(string text)
             {
                 swift = -2;
             }
+            else if (trimmedText.EndsWith("day after"))
```

Comment:
It's best to move these hardcoded strings to the YAML files. It will avoid repeating them for every platform.
And ideally this should be a common method, that each language config could reuse or re-implement.